pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1225,6 +1346,76 @@ func podToMemberPod(pod *v1.Pod, includeIP, includePodRef bool) *controlplane.Gr
 	return memberPod
 }
 
+func externalEntityToGroupMember(ee *v1alpha1.ExternalEntity, includeRef bool) *controlplane.GroupMember {
```

Comment:
do we need includeRef? always true when called

Reply:
Got rid of it.